tests/web/test_main.py

```diff
@@ -516,3 +516,24 @@ def test_get_lineage(web_sushi_context: Context) -> None:
             }
         },
     }
+
+
+def test_table_diff(web_sushi_context: Context) -> None:
+    web_sushi_context.plan(
+        "dev",
+        no_prompts=True,
+        auto_apply=True,
+        skip_tests=True,
+    )
+    response = client.get(
+        "/api/table_diff",
+        params={
+            "source": "prod",
+            "target": "dev",
+            "on": "s.id = t.id",
```

Comment:
while you're add it, can you add the automatic usage of "grain" to the table diff? so there's now a concept of grain which will populate the on field auto, so you won't need to set this manually

Reply:
Yea, I will add it.